docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,112 @@
+reacts to new throttle commands, and the module will eventually :ref:`manual_timeout`.
+
+Now, suppose that you connect DroneCAN as before, but now, also connect your module to a DSHOT output on your flight controller.
+
+.. image:: ../_static/manual_images/redundant_throttle/dshot_and_dronecan_connection.png
+    :width: 50%
+
+Under normal operating conditions, the module will apply only throttle commands received via DroneCAN (as determined by your redundant throttle configuration discussed :ref:`below <redundant_throttle_config>`). 
+If we once again sever the DroneCAN connection, however, the module will seamlessly transition to accepting DSHOT commands. By connecting multiple throttle sources, and 
+leveraging Vertiq's redundant throttle feature, you robustify your vehicle's ability to fly safely even after communication failures.
+
+
+.. _redundant_throttle_config:
+
+***************************************
+Redundant Throttle Configurations
+***************************************
+
+There are two types of configurations available with the redundant throttle. The first defines the amount of time that the module waits before switching to another throttle 
+source if the active source goes offline. The second set of parameters defines the priorities of each supported throttle source. You can find the specific parameters 
+at :ref:`throttle_source_manager`.
+
+Vertiq's supported throttle sources are :ref:`DroneCAN <dronecan_protocol>`, :ref:`IQUART <uart_messaging>`, and :ref:`hobby protocols <hobby_protocol>`. Please note that only priority values corresponding to protocols supported by 
+your module have any effect. See your module's family page to see what throttle sources are supported.
+
+Priorities can be valued from 0 to 3. Setting a priority of 0 indicates that the module will ignore all throttle messages received from the configured source. 
+Priority values [1, 3] define the priority of each protocol against the others where 3 defines the highest priority.
+
+.. note:: 
+    Due to Vertiq modules' hardware, it is not possible to use both IQUART and hobby protocols simultaneously. As such, it is only possible to use DroneCAN with one of IQUART 
+    or hobby as redundant throttle sources.
+
+Configuration Example 1
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- IQUART's priority to 3 and DroneCAN's to 1 (the module will automatically set hobby's to 2 ensuring that there are no two matching priorities)
+- Flight controller outputting both :ref:`IQUART Flight Controller Interface commands <controlling_ifci>` as well as DroneCAN commands
+- Throttle source timeout configured to 0.5 seconds
+
+The sequence diagram below illustrates how your module reacts to received throttle commands as well as how it deals with switching between sources.
+
+.. image:: ../_static/manual_images/redundant_throttle/iquart_prio_example.png
+    :width: 50%
+

```

Comment:
When IQUART disconnects, the flight controller still sends IQUART commands right? It's just that none of the commands will reach the module due to the severed connection. Is performance hindered?

Reply:
It depends on the failure honestly. If the wire just gets cut, then the flight controller will keep sending UART messages...that just won't go anywhere. If there's a software failure or the flight controller's serial peripheral explodes, it could just stop sending IQUART altogether. If your throttle timeout is set to something reasonable, there is no performance impact in this case. The flight controller will output the same throttle value on both IFCI and DroneCAN, so when it switches over to DroneCAN, it'll keep the same command it had from IQUART